File: Fixed Stack Dynamic/ok.java
public class Ok {
    public static void main(String[] args) {
        // Fixed Stack-Dynamic Array
        int[] arr = new int[5]; // Array size is set at compile-time and stored on the stack for primitive types

        // Populate the array with values
        for (int i = 0; i < arr.length; i++) {
            arr[i] = i * 10;
        }

        // Output the array elements
        System.out.println("Fixed Stack-Dynamic Array:");
        for (int num : arr) {
            System.out.print(num + " ");
        }
    }
}

File: Fixed Stack Dynamic/ok.py
# Python lists are inherently dynamic, but we can imitate fixed-size stack behavior

def fixed_stack_dynamic():
    # Simulating a fixed-size, stack-like array using a list
    arr = [0] * 5  # Pre-allocated with fixed size

    # Fill the list with values
    for i in range(len(arr)):
        arr[i] = i * 10

    # Display the list contents
    print("Fixed Stack-Dynamic Array (simulated in Python):")
    print(arr)

fixed_stack_dynamic()

